lib/valkyrie/types.rb

```diff
@@ -37,15 +37,19 @@ module Types
       end
     end
 
+    Array = Dry::Types['coercible.array'].default([].freeze)
+
+    Array = Dry::Types['array'].constructor do |value|
```

Comment:
What does it mean that `Array` is set twice this way?

Reply:
It means I forgot to delete that line. Good catch.